packages/client-account/src/methods/accountCopyIndex.ts

```diff
@@ -58,7 +58,20 @@ export const accountCopyIndex = (
     )
     .then(() => Promise.resolve());
 
-  return createWaitablePromise(promise, () =>
```

Comment:
The problem as on this line, that as resolving the original `promise` only fortting about the promises inside of the `responses` variable.